### ISSUE TITLE
option-t.d.ts: error: Parameter 'T' implicitly has an 'any' type.

### ISSUE THREAD

**Issue by gifnksm** [saneyuki/option-t.js#57]
`tsc --noImplicitAny` causes error with `option-t@0.9.0`.

```
// foo.ts
/// <reference path="./node_modules/option-t/option-t.d.ts" />
```
```
$ tsc --version
message TS6029: Version 1.5.0-beta
$ tsc --noImplicitAny ./foo.ts
node_modules/option-t/option-t.d.ts(30,25): error TS7006: Parameter 'T' implicitly has an 'any' type.
```
https://github.com/saneyuki/option-t.js/blob/master/option-t.d.ts#L30

I'm a newbie of typescript, so, I'm sorry if this is caused by typescript's bugs...

**Comment by saneyuki** [comment]
Thank you for your reporting. This is our d.ts file's bug.

I fixed this by #58, and released a v.0.9.1. please try it :)

**Comment by gifnksm** [comment]
No compile error happens with v0.9.1.
Thank you!
